airflow/www/views.py

```diff
@@ -859,8 +895,12 @@ def rendered(self):
             title=title)
 
     @expose('/get_logs_with_metadata')
-    @has_dag_access(can_dag_read=True)
-    @has_access
+    @auth.has_access([
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAGS),
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_TASK_INSTANCE),
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_LOG),
```

Comment:
```suggestion
```

Reply:
Are you saying that this endpoint shouldn't check for can_read on Logs? Because this is the `get_logs_with_metadata` endpoint